Model/Sync/Customers/Processor.php

```diff
@@ -372,7 +372,7 @@ public function resetCustomerSyncStatus($customerId, $customerStoreId, $value, $
             ];
         }
 
-        $this->insertOnDuplicate('yotpo_customers_sync', $customersSyncData);
+        $this->insertOrUpdateCustomerSyncData($customersSyncData);
```

Comment:
This could've been part of the commit declaring `insertOrUpdateCustomerSyncData`